backend/tracim_backend/lib/core/mention.py

```diff
@@ -166,6 +168,16 @@ def get_receiver_ids(
     def _create_mention_events(
         cls, mentions: typing.Iterable[Mention], content: Content, context: TracimContext
     ) -> None:
+        role_api = RoleApi(session=context.dbsession, config=context.app_config, current_user=None)
+        workspace_members_usernames = [
```

Comment:
You could use a `set()` here, it is quicker than a list for the `in`/`not in` operators.

Reply:
Are you sure about that, especially for small lists? a set has to manage a hash table while a list does not.